### PR TITLE
Fix features page.

### PR DESCRIPTION
**Pull request by XhmikosR**
Use Bootstrap's .thumbnail class to fit the images to the page size.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -90,6 +90,11 @@ a img {
   border: none;
 }
 
+.thumbnail {
+  border: none;
+  box-shadow : none;
```
Comment (nschonni) [inline assets/css/style.css]: Extra space before semicolon

Comment (XhmikosR) [inline assets/css/style.css]: Fixed

**Comment by benbalter** [comment]
Is it possible to post before/after screenshots? (or is the output identical)

**Comment by XhmikosR** [comment]
I can sure; the output isn't identical otherwise there would be nothing to fix :p 

before http://i.imgur.com/xlKCWuy.png
after http://i.imgur.com/0kjE7Jm.png

**Comment by benbalter** [comment]
Awesome. Thanks for this.

**Pull request merged**